### PR TITLE
add initial draft of ADR

### LINKED ISSUE #389: Create ADR noting intentional correspondence of Automatable and Value Density to CVSSv4 Supplemental metrics.
A few SSVC decision points were intentionally matched against CVSS v4 Supplemental Metrics. Specifically:

| SSVC DP | CVSS v4 |
| --- | --- |
| Automatable v2.0.0 | https://www.first.org/cvss/v4.0/specification-document#Automatable-AU |
| Value Density v1.0.0 | https://www.first.org/cvss/v4.0/specification-document#Value-Density-V |

We should record this in a new ADR record.

Possible dependency on
- #311 
If we choose to align Safety to explicitly address IEC 61508, then we might also include SSVC's Public Safety Impact and CVSS v4 Safety as deliberately aligned as well.

### PR DESCRIPTION
**Pull request by j---**
Closes #389 if merged.

### CODE REVIEW

**Comment by j---** [comment]
Can someone help me understand if the linkchecker is failing because the PR needs to be merged before the link can find the file that the PR creates? 

> Error: URLError: <urlopen error [Errno 2] No such file or directory: '/home/runner/work/SSVC/SSVC/site/adr/0011-automatable-and-value-density-and-CVSSv4.md'>

If that were the case, do we override the error?

**Comment by ahouseholder** [comment]
The ADR doc itself isn't in the commit history yet. The only change committed here is to`docs/adr/index.md` so the linkchecker is correctly identifying a 404 on `0011-automatable-and-value-density-and-CVSSv4.md`

File: docs/adr/index.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ the decision records that have been made.
 - [0008 - Decision Points are Ordered Sets](0008-decision-points-are-ordered-sets.md)
 - [0009 - Outcomes are Ordered Sets](0009-outcomes-are-ordered-sets.md)
 - [0010 - Outcome Sets are separate from Decision Point Groups](0010-outcome-sets-are-separate-from-decision-point-groups.md)
+- [0011 - Correspondence between Automatable v2.0.0, Value Density v1.0.0, and CVSS v4](0011-automatable-and-value-density-and-CVSSv4.md)
```
Comment (ahouseholder) [inline docs/adr/index.md:25]: Need to add/commit/push the file that is the target of this link.

**Review by ahouseholder — CHANGES_REQUESTED** [review]

File: docs/adr/index.md (hunk 1 above)
Comment (j---) [reply docs/adr/index.md:25]: Well that's embarrassing. Anyway, fixed.

**Review by cgyarbrough — APPROVED** [review]
This change makes sense and should work. I approve.

**Review by ahouseholder — APPROVED** [review]

**Pull request merged**